emission/core/wrapper/modeprediction.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import emission.core.wrapper.wrapperbase as ecwb
+import emission.core.wrapper.motionactivity as ecwm
+import enum as enum
+
+
+class AlgorithmTypes(enum.Enum):
+	'''
+	We could use NOT_PREDICTED to indicate a prediction that is empty. If a prediction returns zero,
+		it can be incorrectly assumed that the mode predicted it IN_VEHICLE if the prediction returned an error or was never
+		made in the first place. Using NOT_PREDICTED can help distinguish the true zeros from the erroneous ones.
+
+	'''
+	NOT_PREDICTED = 0
+	RANDOM_FOREST = 1
+
+
+class Modeprediction(ecwb.WrapperBase):
+	props = {"trip_id": 	ecwb.WrapperBase.Access.WORM, 	# *******the trip that this is part of
+            "section_id": 	ecwb.WrapperBase.Access.WORM, 	#The section id that this prediction corresponds to
+            "algorithm_id": ecwb.WrapperBase.Access.WORM, 	#The algorithm which made this prediction
+            "confirmed_mode": ecwb.WrapperBase.Access.WORM, #The mode inputted by the user (if it exists)
```

Comment:
Yeah I think it should be a separate wrapper object. But then maybe it should not just be mode, but confirmed everything. When you confirm a trip, you are confirming start/end time, location, etc as well.

Just leave confirmed_mode out for now or leave it in section. This needs some design.